<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>135</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>716</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>716</width>
    <height>135</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="select_csv_Edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>12</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="select_csv_Button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>11</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать csv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_directory_Button">
    <property name="geometry">
     <rect>
      <x>546</x>
      <y>11</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Директория сохранения html</string>
    </property>
   </widget>
   <widget class="QPushButton" name="history_Button">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>История</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="select_directory_Edit">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>12</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_Button">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Преобразовать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csv_info">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>11</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="html_info">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>11</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="history_info">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>17</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
